WEB-MAC-CHAR/src/main/resources/static/js/gameStartSocket.js

```diff
@@ -11,6 +14,12 @@ export default class gameStartSocket {
         this.destinationUrl = destinationUrl;
         stompClient.subscribe(gameStart_url, gameStart => {
             document.querySelector('.player_role_name').textContent = '은 ' + gameStart.body + '입';
+            let playerStatus = document.getElementsByClassName('player_status');
+            Array.from(playerStatus).forEach( entry => {
+              entry.innerText = '';
```

Comment:
how about this?
```js
Array.from(playerStatus).forEach(status => status.textContent='');
```
we don't need to `{}` in arrow function, but `Array.from()` is good!!

Another way,
Add Class `play_not_ready` when game started.